places/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Address(models.Model):
```

Comment:
Tal vez no he sido claro en absoluto sobre como puede ser interesante manejar las direcciones. Si ha sido así, no era mi intención, y espero aclararlo un poco más con la captura de alguien que, en mi opinión, hace un trabajo relativamente bueno al respecto: Amazon (digo bueno porque, en fin, ellos tienen que realmente mandar paquetes, con lo que saben si estos datos son suficientes o no para llegar al usuario).

De aquí, evidentemente el nombre y apellidos se cambia por el nombre del lugar, que no pertenece a la dirección.

Lo que decía además es que es interesante tener un modelo de ciudades fuera de la dirección, ya que ciudad y país suelen ir juntos, y para poder tener más fácil una consulta del tipo "¿cuántos usuarios tengo registrados que sean de Santiago"? (¿Santiago... de Chile o de España? Bueno, si la ciudad tiene su propio modelo, lo que buscas es aquellos objetos con la clave foránea 27). Pero vamos, que eso no tengo una opinión fuerte al respecto.

Lo que sí opino es:
* Que el país debería de ser el código de dos letras del mismo (esto es un API, queremos datos normalizados, *si se puede*)
* Que el número/letra/puerta/escalera/bloque/punto kilométrico y otras nociones de situación del edificio y la vivienda dentro de un municipio son demasiado dependientes de las costumbres del lugar

Para lo demás, me gustaría poder *hablar* via comentarios por aquí mismamente al respecto, y poder llegar a algo razonable.

![screenshot from 2016-09-09 15-27-11](https://cloud.githubusercontent.com/assets/581152/18388450/fc9409a6-76a1-11e6-8366-87d16f7a571b.png)